app/api/schema/tax.py

```diff
@@ -26,30 +26,20 @@ class Meta:
                          type_='event')
 
 
-class TaxSchema(Schema):
+class TaxSchema(TaxSchemaPublic):
```

Comment:
Shouldn't TaxSchemaPublic inherit from TaxSchema instead of otherwise?

Reply:
No, TaxSchemaPublic will contain lesser number of fields. We don't want to show everything to the public. TaxSchema is for the event organizer/admin so we will show everything to him/her.
For e.g.-
Lets say a ExampleSchema has 4 fields A,B,C,D. 2 of them(C,D) are sensitive and we only want to show to the organizer or the admin, not to any one else. A and B however aren't sensitive and can be shown to general public.

So ExampleSchemaPublic(Schema):
      A
      B

ExampleSchema(ExampleSchemaPublic):
      C
      D

Clear ?

Reply:
I guess usually on naming notation the shorter name is usually more general, so normally you'd do something like ExampleSchema for the public and ExampleSchemaPrivate for the more special case, but it doesn't matter very much. Yes, it's clear.

Reply:
Yeah but I am trying to follow the way it has been done in other sections of the project. If you've any other suggestions please suggest otherwise please approve :)